examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  neutron:
+    template:
+      customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        service_plugins = qos,ovn-router,trunk,segments,port_forwarding,log
+        dhcp_agent_notification = false
+        agent_down_time = 600
+        router_distributed = true
+        router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+        allow_automatic_l3agent_failover = true
+        debug = true
+        log_dir = /var/log/neutron
+        default_availability_zones = zone-1,zone-2
```

Comment:
IIRC,  for this (default_availability_zones = zone-1,zone-2) to work properly it will require also settings in ovs db on compute nodes

Reply:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/573